docs/reference/job-reference.md

```diff
@@ -342,24 +344,22 @@ plot_results(
     figsize,
     bands,
     titles,
-    filpaths,
+    filepaths,
     plot_file_format,
     kwargs,
```

Comment:
@saad-up42 Can you please update `kwargs` -> `**kwargs` in your PR? In my [PR](https://github.com/up42/up42-py/pull/487/files#:~:text=kwargs%2C-,**kwargs%2C,-), I ended up on a deep dive of kwargs vs **kwargs, and [found](https://docs.python.org/3/glossary.html#term-parameter) we should be using **kwargs.

Reply:
Interesting.

Done, thanks!